src/components/project-filter/project-filter-directive.tpl.html

```diff
@@ -1,23 +1,23 @@
 <li class="dropdown" refresh-on="OrganizationChanged ProjectChanged" refresh-action="vm.get()" refresh-throttle="5000">
   <a class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" refresh-on="filterChanged" refresh-action="vm.update()">
-    {{vm.filteredDisplayName}} <span class="caret"></span>
+    {{vm.filteredDisplayName | translate}} <span class="caret"></span>
   </a>
   <ul class="dropdown-menu w-xl scrollable-menu" ng-style="{'max-height': vm.filterDropDownMaxHeight + 'px'}" auto-active>
-    <li><a ng-href="{{vm.urls.projects.all}}">All Projects</a></li>
+    <li><a ng-href="{{vm.urls.projects.all}}">{{'All Projects' | translate}}</a></li>
     <li role="separator" class="divider"></li>
-    <li ng-if="vm.isLoadingOrganizations" disabled><a href="#">Loading Organizations...</a></li>
+    <li ng-if="vm.isLoadingOrganizations" disabled><a href="#">{{'Loading Organizations...' | translate}}</a></li>
```

Comment:
I wonder on somethings like this if it makes sense to split out the translation. For example we have translations for `Loading...` and now `Loading Organizations...`. Do you think it would make sense to do Translations for only the root word `Loading` and then you'd split this into transitions for `Loading` and `Organizations`. I think this makes sense but not sure how some languages might reverse the words or change wording based on multiple words `Loading Organizations`

Reply:
OK.Thanks for your suggest. I will modify `Loading...` to `Loading` and `Loading Organizations...` to `{{Loading | translate}} {{'Organizations'|translate}}`.